.travis.yml

```diff
@@ -3,8 +3,8 @@ dist: trusty
 sudo: false
 group: beta
 scala:
-- 2.12.1
-- 2.11.8
+- 2.12.6
```

Comment:
Scala 2.12.8 is available already. Please update.

Reply:
I've committed a change with all the version changes